frontend/src/stores/MainStore.js

```diff
@@ -316,10 +318,14 @@ class MainStore {
     @observable toastHeader = "";
     @observable toastMessage = "";
     @action.bound downloadReport(url) {
+        this.showModal = false;
         let apiUrl = (apiUrl = this.config[url]);
         if (this.canEdit) {
             apiUrl = `${apiUrl}?editKey=${this.config.editSettings.editKey}`;
         }
+        if (url === "wordUrl") {
```

Comment:
no string w/ +; use the formatting with backticks, this is consistent wit the canEdit line above

Reply:
i replaced string concat to using backticks.